.github/workflows/uki.yaml

```diff
@@ -11,125 +11,22 @@ concurrency:
-          sudo apt-get remove -y hhvm || true
-          sudo apt-get remove -y powershell || true
-          sudo apt-get remove -y firefox || true
-          sudo apt-get remove -y monodoc-manual || true
-          sudo apt-get remove -y msbuild || true
-          sudo apt-get remove -y microsoft-edge-stable || true
-          sudo apt-get remove -y '^google-.*' || true
-          sudo apt-get remove -y azure-cli || true
-          sudo apt-get remove -y '^mongo.*-.*|^postgresql-.*|^mysql-.*|^mssql-.*' || true
-          sudo apt-get remove -y '^gfortran-.*' || true
-          sudo apt-get autoremove -y
-          sudo apt-get clean
-          echo
-          echo "Listing top largest packages"
-          pkgs=$(dpkg-query -Wf '${Installed-Size}\t${Package}\t${Status}\n' | awk '$NF == "installed"{print $1 "\t" $2}' | sort -nr)
-          head -n 30 <<< "${pkgs}"
-          echo
-          sudo rm -rfv build || true
-          df -h
-      - uses: actions/checkout@v4
-      - run: |
-          git fetch --prune --unshallow
-      - name: Release space from worker
-        run: |
-          sudo rm -rf /usr/local/lib/android # will release about 10 GB if you don't need Android
-          sudo rm -rf /usr/share/dotnet # will release about 20GB if you don't need .NET
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@master
-        with:
-          platforms: all
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@master
-      - name: Install Cosign
-        uses: sigstore/cosign-installer@main
-      - name: Login to Quay Registry
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
-        run: echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USERNAME }} --password-stdin quay.io
-      - name: Install earthly
-        uses: Luet-lab/luet-install-action@v1.1
-        with:
-          repository: quay.io/kairos/packages
-          packages: utils/earthly
-      - name: Block all traffic to metadata ip  # For cloud runners, the metadata ip can interact with our test machines
-        run: |
-          sudo iptables -I INPUT -s 169.254.169.254 -j DROP
-          sudo iptables -I OUTPUT -d 169.254.169.254 -j DROP
-      - name: Build 🔧
-        run: |
-          earthly --platform=linux/amd64 +base-image \
-            --VARIANT=core \
-            --FLAVOR=${{ matrix.flavor }} \
-            --FLAVOR_RELEASE="${{ matrix.flavor_release }}" \
-            --BASE_IMAGE=${{ matrix.base_image }} \
-            --MODEL=generic \
-            --FAMILY=${{ matrix.family }} \
-            --BOOTLOADER=systemd-boot
-          sudo mv build/* .
-          sudo rm -rf build
-      - name: Install kairos-agent (for versioneer)
-        uses: Luet-lab/luet-install-action@v1.1
-        with:
-          repository: quay.io/kairos/packages
-          packages: system/kairos-agent
-      - name: Push to quay
```

Comment:
I think we still need core because this is what the edge team has been using for their tests, no?

Reply:
Core was just the same as the Ubuntu lts no? It's building 24.04? Or did I screwed it up?

Reply:
Ohh this pushes it to quay on master!

Good catch, I thought we had that on the reusable job...we may need to have that there anyway?

Reply:
No I think we don't have it on the reusable job but I guess we could use the reusable job to build and then still push just on the ones we want to push, since we don't want fedora atm for example

Reply:
I added it to the reusable job, now it pushes AFTER tests pass and only if its from master and UBUNTU

I would change that to push always, so any image that we test is pushed....makes sense to me, if we run tests and it passes, we should make that image available no?

Reply:
well, maybe not unless we "support" them somehow rigth?

Its nice to have the images pushed from master so we can test building isos from there and such. But on release, we are committing to it rigth? like we are saying is supported by making it an official release artifact rather than a in-progress dev version. I would be ok with that but our current focus is on ubuntu so not sure if it would make sense to make it officially supported with our bandwidth :(

Something to discuss on Monday I guess?